lib/kitchen/driver/oci/blockstorage.rb

```diff
@@ -131,14 +131,25 @@ def attachment_response(attachment_id)
         end
 
         def volume_details(volume)
-          OCI::Core::Models::CreateVolumeDetails.new(
+          details = OCI::Core::Models::CreateVolumeDetails.new(
             compartment_id: oci.compartment,
             availability_domain: config[:availability_domain],
             display_name: volume[:name],
```

Comment:
Aesthetically, I'd like the cloned volume to retrieve the display name from the volume being cloned and append `(Clone)` to the display name similar to how I am doing the boot volume cloning [here](https://github.com/stephenpearson/kitchen-oci/blob/73963247dd7c47564f72c6528e1c424123f64a60/lib/kitchen/driver/oci/models/compute.rb#L115)

Reply:
Sure, something like this can be done but I think it depends on how we decide to split the clone volume code from the regular volume creation as mentioned in the above suggestion. 

```ruby
def add_volume_source_details(details, volume)
  if volume.key?(:source_details)
    details.display_name = clone_volume_display_name(volume[:source_details][:id])
    details.source_details = OCI::Core::Models::VolumeSourceFromVolumeDetails.new(
      id: volume[:source_details][:id]
     )
  end
  details
end
        
def clone_volume_display_name(volume_id)
  "#{api.blockstorage.get_volume(volume_id).data.to_hash[:displayName]} (Clone)"
end
 ```

Reply:
I updated the code now, it does as you asked:

```
-----> Creating <default-ubuntu-2204>...
       Waiting for SSH service on <IPADDRESS>:22, retrying in 3 seconds
       [SSH] Established
       Creating <data (Clone)>...
       Finished creating <data (Clone)>.
       Attaching <data (Clone)>...
       Finished attaching <data (Clone)>.
       Creating <test-jose-volume-3 (Clone)>...
       Finished creating <test-jose-volume-3 (Clone)>.
       Attaching <test-jose-volume-3 (Clone)>...
       Finished attaching <test-jose-volume-3 (Clone)>.
       Finished creating <default-ubuntu-2204> (1m54.67s).
-----> Test Kitchen is finished. (1m55.77s)
```